packages/vue-query/src/useQueries.ts

```diff
@@ -155,7 +155,7 @@ export function useQueries<
     ...options
   }: {
     queries: MaybeRefDeep<UseQueriesOptionsArg<T>>
-    combine?: (result: UseQueriesResults<T>) => TCombinedResult
+    combine?: (result: UseQueriesResults<T>, combinedResult: TCombinedResult | undefined) => TCombinedResult
```

Comment:
you'd also need to change the interface of `useQueries` in the react adapter please